README.md

```diff
@@ -173,7 +173,7 @@ Note: Upstart/SysV init based OS types are not supported.
   - [kube-ovn](https://github.com/alauda/kube-ovn) v1.11.5
   - [kube-router](https://github.com/cloudnativelabs/kube-router) v2.0.0
   - [multus](https://github.com/k8snetworkplumbingwg/multus-cni) v3.8
-  - [weave](https://github.com/weaveworks/weave) v2.8.1
+  - [ReWeave](https://github.com/rajch/weave) v2.8.7
```

Comment:
`weave` change to community version. How do I add a note?

Reply:
Will this version be better?
```suggestion
  - [weave](https://github.com/rajch/weave) (community version) v2.8.7
```

Reply:
Maybe reach out to the project to know if they have preferences in how they are designated / referenced ?

Reply:
Related issue: https://github.com/rajch/weave/issues/5

Reply:
He has replied.